ably/ably-ui
# [WEB-2568] Add live updates link to meganav

Jira Ticket Link / Motivation
----
https://ably.atlassian.net/browse/WEB-2568



Summary of changes
----
added item to user cases menu



How do you manually test this?
----
check the menu in the preview app and verify that element is presented in that app


Reviewer Tasks (optional)
----




Merge/Deploy Checklist
----




Frontend Checklist
----
